### ISSUE TITLE
Include rx-datascript?

### ISSUE THREAD

**Issue by kylecordes** [typeetfunc/datascript-mori#7]
This is a (small?) feature request.

Currently a compelling benefit of datascript-mori is that it includes a single copy of the CLJS runtime, and the two libraries you are most likely to need while working with Datascript from JavaScript.

The feature request is to include other adjacent, small (and therefore pretty painless to people not using them) libraries, to further expand the set of related tools trivially loadable for use in a JavaScript project.

How about including rx-datascript? It's essentially one source file of 108 lines.

https://github.com/typeetfunc/rx-datascript

(Obviously there are diminishing returns past a very small number of related libraries. Someone needs more than a small handful probably has to make their own bundle.)

**Comment by typeetfunc** [comment]
Hi! Thanks for issue - this is cool idea :)
But this will be possible only after the `rx-datascript` becomes stream-library agnostic(as in [recompose](https://github.com/acdlite/recompose/blob/master/src/packages/recompose/setObservableConfig.js)) and independent of `rxjs`
(https://github.com/typeetfunc/rx-datascript/blob/master/src/index.js#L2).
Rx.js is very big and fat dependence. And it is not needed for most of users `datascript-mori`.

**Comment by kylecordes** [comment]
Yes, thank you, this makes sense. It is frustrating to see each library "out there" that wants to provide observability, have to take on such a large dependency.
